### ISSUE TITLE
Email is getting garbled with flash messages

### ISSUE THREAD

**Issue by chrislerum** [rubysherpas/forem#614]
this from the heroku demo site

https://www.dropbox.com/s/d49g1sgb2y6eqyw/Screenshot%202014-10-29%2016.37.27.png?dl=0

**Comment by jengel3** [comment]
The bootstrap site is pretty broken right now, needs some work I suppose.

**Comment by radar** [comment]
Yup, it definitely needs some work. I am too busy working on updating Rails 4 in Action in my (tiny amounts of) spare time, so someone will need to take the lead on this.
